reference/constraints/Cascade.rst

```diff
@@ -0,0 +1,120 @@
+Cascade
+=======
+
+.. versionadded:: 5.2
+
+    The :class:`Symfony\\Component\\Validator\\Constraints\\Cascade` was
+    introduced in Symfony 5.2 and requires PHP 7.4.
```

Comment:
What do you think about using a caution block for PHP version restriction ? I think this information is more important

Reply:
Actually, versionadded blocks are removed by maintainers when upmerging the branch. Caution blocks are not, and as this is only a relevant info for 5.4, I would say it should stay in this block 🙂